### ISSUE TITLE
This project seems unmaintained any alternatives?

### ISSUE THREAD

**Issue by joshbedo** [stereobooster/react-snap#514]
Just noticed that my meta tag descriptions weren't updating correctly and this project doesn't seem to be active anymore. Anyone know of good alternatives?

**Comment by joshbedo** [comment]
Actually just looked into generated build and it looks like its adding its own meta tag and then adding the correct react-helmet meta tag the problem is its adding both.

**Comment by ricky11** [comment]
did you fix this? yes it looks like there isn't a maintainer anymore.. too bad as it was a great package..

**Comment by ml242** [comment]
This is too good to be abandoned!

**Comment by ml242** [comment]
sadly i had to move my project to a next.js build because i couldn't iron out all of the issues i had here. i stand by my former comment though, it's a cool project and would have saved time.

**Comment by ricky11** [comment]
@ml242 Would like to know how you added nuxt static site generation to an existing vue project.  Did you need to amend a lot of the folder structure in order to incorporate nuxt?

**Comment by karlhorky** [comment]
> sadly i had to move my project to a next.js build because i couldn't iron out all of the issues i had here

Yeah I think moving to Next.js is a good option for `create-react-app` projects depending on `react-snap`, since it does seem unmaintained.
